lib/main_micro_alp.pm

```diff
@@ -266,7 +266,30 @@ sub load_journal_check_tests {
     loadtest 'shutdown/shutdown';
 }
 
+sub load_slem_on_pc_tests {
+    my $args = OpenQA::Test::RunArgs->new();
+
+    loadtest("boot/boot_to_desktop");
+    loadtest("publiccloud/prepare_instance", run_args => $args);
+    loadtest("publiccloud/registration", run_args => $args);
+    loadtest("publiccloud/ssh_interactive_start", run_args => $args);
+    loadtest("publiccloud/instance_overview", run_args => $args);
+    loadtest("publiccloud/slem_prepare", run_args => $args);
+
+    if (get_var("PUBLIC_CLOUD_CONTAINERS")) {
```

Comment:
I would turn this block into single line:
`load_container_tests() if is_container_test`

`load_container_tests()` will fail if `CONTAINER_RUNTIME` is not defined.

Reply:
I would agree with you, if we would not need the `return 1` in the same block.

Reply:
Sorry, I was looking at the wrong block. But still, silently not executing the tests is IMHO worse than immediately dying with an error message.

Then again, those error messages are anyways replaced by the generic "something something went wrong in main.pm, please check syntax", so I guess in the end it doesn't make a difference.

Long story short: Will amend your suggestion.

Reply:
Amended, new VR: https://duck-norris.qe.suse.de/tests/12588